src/Hooks/CommandStartingListener.php

```diff
@@ -39,17 +40,12 @@ public function __construct(
 
     public function __invoke(CommandStarting $event): void
     {
-        try {
-            if ($this->nightwatch->state->name === null) {
-                $this->nightwatch->state->name = $event->command;
-                $this->nightwatch->state->executionPreview = Str::tinyText($event->command);
-            } else {
-                return;
-            }
-        } catch (Throwable $e) {
-            $this->nightwatch->report($e);
```

Comment:
I've moved this work down so that we configure the sampling rate before we do this work. This allows us to bypass the work when we don't need to do it.

Reply:
Love this!